packages/components/psammead-navigation/src/index.jsx

```diff
@@ -109,43 +125,94 @@ const CurrentLink = ({ children: link, script, currentPageText }) => (
   </>
 );
 
-export const NavigationUl = ({ children, ...props }) => (
-  <StyledUnorderedList role="list" {...props}>
-    {children}
+CurrentLink.propTypes = {
+  children: string.isRequired,
+  script: shape(scriptPropType).isRequired,
+  currentPageText: string,
+};
+
+CurrentLink.defaultProps = {
+  currentPageText: null,
+};
+
+export const NavigationUl = ({ children, isSwipeable, ...props }) => (
+  <StyledUnorderedList role="list" isSwipeable={isSwipeable} {...props}>
+    {React.Children.map(children, child =>
+      React.cloneElement(child, { isSwipeable }),
+    )}
   </StyledUnorderedList>
 );
 
+NavigationUl.propTypes = {
+  children: node.isRequired,
+  isSwipeable: bool,
+};
+
+NavigationUl.defaultProps = { isSwipeable: false };
+
 export const NavigationLi = ({
   children: link,
   url,
   script,
   currentPageText,
   active,
   service,
+  dir,
+  isSwipeable,
   ...props
-}) => (
-  <StyledListItem role="listitem">
-    {active && currentPageText ? (
-      <StyledLink
-        href={url}
-        script={script}
-        currentLink="true"
-        service={service}
-        {...props}
-      >
-        <CurrentLink script={script} currentPageText={currentPageText}>
+}) => {
+  const tabIndex = isSwipeable && { tabIndex: -1 };
```

Comment:
```suggestion
  const tabIndex = isSwipeable &&  -1;
```
`tabIndex` is already defined as a variable. We don't need to create an object with `tabIndex` as a key again IMO.

Reply:
@DenisHdz can we revert this change.  Applying my suggestion leads to console warnings which we have been trying to get rid off in both `psammead` and `simorgh`

<img width="553" alt="Screenshot 2019-11-24 at 16 52 27" src="https://user-images.githubusercontent.com/48688870/69495754-74e4cd80-0edb-11ea-84de-eb3bf5301cff.png">

Reply:
Sure. I initially did it in that way to just add `tabIndex` to the DOM when `isSwipeable` is true.